src/collections/_documentation/workflow/integrations/legacy-integrations.md

```diff
@@ -117,21 +117,24 @@ heroku labs:enable runtime-dyno-metadata -a <app name>
 
 This exposes the `HEROKU_SLUG_COMMIT` environment variable, which most Sentry SDKs will automatically detect and use for configuration.
 
-Next you’ll want to add your repository and setup a deploy hook.
+Next, you’ll want to add your repository and setup a deploy hook:
 
-1.  Start by connecting your repository to your Sentry organization so that we can automatically retrieve your commit data.
+1.  Start by connecting your repository to your Sentry organization so that we can automatically retrieve your commit data. To do this find your repository integration (e.g. GitHub, GitLab, Bitbucket, etc.) in Settings > Integrations and from the "Configurations" tab click "Configure".
 
-    [{% asset add-repo.png %}]({% asset add-repo.png @path %})
+    [{% asset heroku-configure-repo-integration.png %}]({% asset heroku-configure-repo-integration.png @path %})
     
-2.  Enable the Heroku integration in your Sentry Project Settings.
+    On the configure page, click "Add Repository" and select the repository you want to use.
+    
+    [{% asset heroku-repo-add.png %}]({% asset heroku-repo-add.png @path %})
+2.  Find the Heroku integration in Settings > Integrations, and click "Add to Project" and select the project you want to use with Heroku.
```

Comment:
Bold the menu path

Also, click...then select (else it seems as if you must do both at the same time)